addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -148,37 +161,33 @@ var QuestionFormWidget = publicWidget.Widget.extend({
      */
     _createOrUpdateQuestion: async function (options) {
         var $form = this.$('form');
+        var validationError = this.$('.o_wslides_js_quiz_validation_error');
```

Comment:
Why did you change the existing function? Any specific reason?

Reply:
The purpose behind editing the existing function was to validate if the function has executed successfully or not.
The modified function returns `true` only when it has successfully created a new question. IMO this refactor enhances the performance as well as the overall utility of the function.

Reply:
@shut-odoo I don't think changing the existing code is a good idea can you achieve this with a different approach?

Reply:
Hello @ibo-odoo :wave: ,
Thanks for the concern :slightly_smiling_face: ,

Following our internal discussions, In the new approach, I have verified the presence of the `o_wslides_js_quiz_validation_error` class.